year-pages/2020.md

```diff
@@ -2,6 +2,7 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [Cloud Community Days](https://ccdays.konfhub.com/) | 17-19 Jun | Online | Ride the Cloud Revolution | No |
```

Comment:
```suggestion
| [Cloud Community Days](https://ccdays.konfhub.com/) | 17-19 Jun | Online | Ride the wave of latest and best in Containers, Serverless, AI & ML, and more. | No |
```

Reply:
okay, i will do.

Reply:
@dharmendranegi I already made the changes (turns out GitHub allows maintainers to do that!) and the code has been merged. :)

Reply:
Thanks.